src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3353,7 +3353,7 @@ namespace InternalHeatGains {
 
 				//For predefined tabular reports related to outside air ventilation
 				ZonePreDefRep( NZ ).isOccupied = true; //set flag to occupied to be used in tabular reporting for ventilation
-				ZonePreDefRep( NZ ).NumOccAccum += NumberOccupants *  Zone( NZ ).Multiplier * Zone( NZ ).ListMultiplier * TimeStepZone;
+				ZonePreDefRep( NZ ).NumOccAccum += NumberOccupants * TimeStepZone;
```

Comment:
I'd have to double check, but I do recall seeing the multiplied people in this table when the note said it should be for a single zone. I don't recall seeing other problems with this table but would have to check again. I think these files are on my laptop at home.